ghdocs/VISUALTESTS.md

```diff
@@ -0,0 +1,70 @@
+# Testing
+
+## Basics
+
+Visual testing is still in the early stages and is subject to change.
+
+Our tests are visual tests are built with with [PhantomCSS](https://github.com/Huddle/PhantomCSS) which makes extensive use of [Casperjs](http://casperjs.org/).
+
+In order to have PhantomCSS run you must have python version 2.6 or higher installed.
+
+To run tests:
+
+1. Navigate to git/office-ui-fabric-react/packages/office-ui-fabric-react in command prompt.
```

Comment:
@joschect can you change this to just `packages/office-ui-fabric-react`. Rather not assume everyone is putting things into their git folder, or naming them the same.